yaml_edit/test/naughty_test.dart

```diff
@@ -21,11 +23,15 @@ void main() {
   for (final string in problemStrings) {
     test('expect string $string', () {
       final doc = YamlEditor('');
-
-      expect(() => doc.update([], string), returnsNormally);
-      final value = doc.parseAt([]).value;
-      expect(value, isA<String>());
-      expect(value, equals(string));
+      runZoned(() {
+        expect(() => doc.update([], string), returnsNormally);
+        final value = doc.parseAt([]).value;
+        expect(value, isA<String>());
+        expect(value, equals(string));
+      },
+          zoneSpecification: ZoneSpecification(
+              print: (Zone self, ZoneDelegate parent, Zone zone,
```

Comment:
Can we leave a comment here, that we are doing this to throw away warnings from `package:yaml`.